addons/l10n_pl_jpk/views/account_move_views.xml

```diff
@@ -7,13 +7,13 @@
         <field name="arch" type="xml">
             <field name="invoice_date" position="after">
                 <field name="l10n_pl_show_delivery_date" invisible="1"/>
-                <field name="l10n_pl_delivery_date" string="Delivery Date" attrs="{'invisible': [('l10n_pl_show_delivery_date', '=', False)]}"/>
+                <field name="l10n_pl_delivery_date" string="Delivery Date" attrs="{'invisible': ['|', ('country_code', '!=', 'PL'), ('l10n_pl_show_delivery_date', '=', False)]}"/>
```

Comment:
Why isn't `l10n_pl_show_delivery_date` enough here ?

Reply:
It is :facepalm: